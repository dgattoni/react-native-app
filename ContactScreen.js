import React from 'react';
import { StyleSheet, Text, View } from 'react-native';

export default class ContactScreen extends React.Component {

  render() {
    const { width } = this.props;
    const styles = StyleSheet.create({
      container: {
        flex: 0,
        width: width,
        backgroundColor: 'yellow',
        alignItems: 'center',
        justifyContent: 'flex-start',
        borderWidth: 5,
        borderColor: 'orange',
        borderStyle: 'solid'
      },
    });

    return (
      <View style={styles.container}>
        <Text>Contact</Text>
        <Text>lorem ipsum</Text>
      </View>
    );
  }
}
