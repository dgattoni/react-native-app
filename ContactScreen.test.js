import React from 'react';
import ContactScreen from './ContactScreen';
import renderer from 'react-test-renderer';

describe('ContactScreen', () => {

  it('displays ContactScreen component', () => {
    const rendered = renderer.create(<ContactScreen />).toJSON();
    expect(rendered).toMatchSnapshot();
  })
});
