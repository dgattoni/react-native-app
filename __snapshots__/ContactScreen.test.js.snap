// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactScreen displays ContactScreen component 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "yellow",
      "borderColor": "orange",
      "borderStyle": "solid",
      "borderWidth": 5,
      "flex": 0,
      "justifyContent": "flex-start",
      "width": undefined,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Contact
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    lorem ipsum
  </Text>
</View>
`;
